src/providers/bedrock.ts

```diff
@@ -553,20 +628,23 @@ export class AwsBedrockCompletionProvider extends AwsBedrockGenericProvider impl
         error: `API call error: ${String(err)}`,
       };
     }
-    logger.debug(
-      `\tAmazon Bedrock API response: ${JSON.stringify(JSON.parse(response.body.transformToString()))}`,
-    );
+    logger.debug(`\tAmazon Bedrock API response: ${response.body.transformToString()}`);
     if (isCacheEnabled()) {
       try {
-        await cache.set(cacheKey, response.body.transformToString());
+        await cache.set(cacheKey, new TextDecoder().decode(response.body));
```

Comment:
this change is pulled from: https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javascriptv3/example_code/bedrock-runtime/models/metaLlama/llama2/invoke_model_quickstart.js#L45